src/test/scala/ru/dgis/reactivezmq/Examples.scala

```diff
@@ -14,7 +14,7 @@ object Examples extends App {
   val socket = context.socket(ZMQ.PULL)
   socket.setReceiveTimeOut(1000)
 
-  val source = ZMQSource(socket, List("tcp://127.0.0.1:12345"))
+  val source = ZMQSource(() => socket, List("tcp://127.0.0.1:12345"))
```

Comment:
i believe we should rewrite this example in order to show that you musn't reuse the same socket instance among materialized graphs